exercises/exercises.csproj

```diff
@@ -38,7 +38,7 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs" Exclude="sgf-parsing\SgfParsing.cs" />
+    <Compile Include="**\*.cs" Exclude="sgf-parsing\SgfParsing.cs;markdown\Markdown.cs" />
```

Comment:
Now I can see it. The reason this file is excluded is simple: we otherwise have two files (the example and the default implementation) with the same namespace and class name, which is a compile error.